app/controllers/railway_stations_controller.rb

```diff
@@ -1,72 +1,46 @@
 class RailwayStationsController < ApplicationController
   before_action :set_railway_station, only: [:show, :edit, :update, :destroy]
 
-  # GET /railway_stations
-  # GET /railway_stations.json
   def index
     @railway_stations = RailwayStation.all
   end
 
-  # GET /railway_stations/1
-  # GET /railway_stations/1.json
   def show; end
 
-  # GET /railway_stations/new
   def new
     @railway_station = RailwayStation.new
   end
 
-  # GET /railway_stations/1/edit
   def edit; end
 
-  # POST /railway_stations
-  # POST /railway_stations.json
   def create
     @railway_station = RailwayStation.new(railway_station_params)
-
-    respond_to do |format|
-      if @railway_station.save
-        format.html { redirect_to @railway_station, notice: 'Railway station was successfully created.' }
-        format.json { render :show, status: :created, location: @railway_station }
-      else
-        format.html { render :new }
-        format.json { render json: @railway_station.errors, status: :unprocessable_entity }
-      end
+    if @railway_station.save
+      redirect_to @railway_station, notice: 'Railway station was successfully created.'
+    else
+      render :new
     end
   end
 
-  # PATCH/PUT /railway_stations/1
-  # PATCH/PUT /railway_stations/1.json
   def update
-    respond_to do |format|
-      if @railway_station.update(railway_station_params)
-        format.html { redirect_to @railway_station, notice: 'Railway station was successfully updated.' }
-        format.json { render :show, status: :ok, location: @railway_station }
-      else
-        format.html { render :edit }
-        format.json { render json: @railway_station.errors, status: :unprocessable_entity }
-      end
+    if @railway_station.update(railway_station_params)
+      redirect_to @railway_station, notice: 'Railway station was successfully updated.'
+    else
+      render :edit
     end
   end
 
-  # DELETE /railway_stations/1
-  # DELETE /railway_stations/1.json
   def destroy
     @railway_station.destroy
-    respond_to do |format|
-      format.html { redirect_to railway_stations_url, notice: 'Railway station was successfully destroyed.' }
-      format.json { head :no_content }
-    end
+    redirect_to railway_stations_url, notice: 'Railway station was successfully destroyed.'
   end
 
   private
 
-  # Use callbacks to share common setup or constraints between actions.
```

Comment:
Why do you remove comments ?

Reply:
> Если что-то уже сделано через scaffolding, нужно "вычистить" контроллеры от ненужного.
I think, it is unnecessary, because scaffold generate them.

Reply:
ok, as you wish.